### ISSUE TITLE
Dragging "nothing" with mouse gives empty screen

### ISSUE THREAD

**Issue by AnddyAnddy** [haxball/haxball-issues#2025]
![image](https://github.com/haxball/haxball-issues/assets/38374141/e0f27463-e658-4d64-8bdf-c1a315e6c9a7)

Opera gx, fps limit disabled, angle d3d9, low latency canvas on

When you drag with the mouse on the game canvas for some seconds, it provides this screen, which can only by toggling the low latency canvas.

I think this bug has already happened with me messing with keys, I probably toggled a shortcut but this happens also when you're trying to drag the mouse, either for nothing or to copy a text

**Comment by ghost** [comment]
I have experienced the same or a similar issue multiple times while using Brave Browser with different Shields settings. The game screen simply stops rendering itself.

**Comment by basro** [comment]
@AnddyAnddy I've tried to reproduce this by dragging the mouse on opera gx with those settings but it doesn't happen to me. How consistently are you able to reproduce this issue?

@yannsan were you also using fps limit disabled?

**Comment by ghost** [comment]
No, I wasn't. Sorry for not mentioning it before. Low-latency canvas enabled, Vulkan enabled (if this helps somehow).
Re-enabling low-latency canvas makes it render again as Anddy said.

**Comment by AnddyAnddy** [comment]
It happens every time, it's not random. Precisely what i'm doing:
- Start a game
- Hold left click with the mouse somewhere in the game canvas
- Keep moving it somewhere else, still held and keep it for some seconds (i'd say 2-3)

https://i.imgur.com/U8QeMxR.mp4
(When you see the mouse, i'm left clicking and holding it)

**Comment by AnddyAnddy** [comment]
I just tried with the low latency canvas disabled, the bug does not happen.

**Comment by basro** [comment]
@AnddyAnddy could you check if you can still reproduce it if you don't use frame rate limit disabled?

**Comment by AnddyAnddy** [comment]
> @AnddyAnddy could you check if you can still reproduce it if you don't use frame rate limit disabled?

I tested mulitple combinations, different browser (chrome), I confirm that the only combination where I could reproduce this bug is Opera gx, fps limit disabled, angle d3d9, low latency canvas on

**Comment by AnddyAnddy** [comment]
Some values:
Opera GX version: LVL5 (core: 100.0.4815.44)
System : Windows 10 64-bit
Chromium version :114.0.5735.199

**Comment by basro** [comment]
could you check if there are any errors logged in the browser inspector console when the bug happens?

**Comment by ghost** [comment]
I wish I had more information. It happened randomly to me a few times after coming back from another tab or program. No heavy usage of computer resources in this case. The chat and room menu were still working.
